source/_posts/rails-docker.md

```diff
@@ -0,0 +1,137 @@
+---
```

Comment:
1-2 long   tail keyword should be there if possible

Reply:
We have two keywords.